lib/worker/batcher/state.ex

```diff
@@ -5,23 +5,23 @@ defmodule BorsNG.Worker.Batcher.State do
   and emits a batch state.
   """
 
-  @typep n :: BorsNG.Database.Status.state_n
-  @typep t :: BorsNG.Database.Status.state
+  @typep status :: BorsNG.Database.Status.t
```

Comment:
this one was incorrect. `state_n` is how the `state` was being stored in the DB. The thing that the function receives is actually a list of `%Status{}` as can be seen